helpers/database-import-scripts/rnacentral/generate_rnacentral_json.py

```diff
@@ -428,16 +507,30 @@ def get_project_accession(biosample):
 
 
 def load_xml(sample_id, insdc_accession=None):
+    if sample_id in ERROR_404:
+        logging.warning("Not attempting to retrieve xml for sample {}. Reason: sample is not in ENA".format(
+            sample_id))
+        return None
     xml_url = 'https://www.ebi.ac.uk/ena/browser/api/xml/{}'.format(sample_id)
     try:
         r = run_xml_request(xml_url)
     except:
-        if insdc_accession.startswith("C"):
-            logging.error("Unable to get XML for sample {}, ENA genome {}".format(sample_id, insdc_accession))
-            sys.exit()
+        if r is None:
+            if insdc_accession.startswith("C"):
```

Comment:
Why "C"?

Reply:
Good point. This was meant to check if the accession belongs to a wgs_set but they can now start with other letters too. Modified the code here https://github.com/EBI-Metagenomics/genomes-catalogue-pipeline/commit/0a4d6a5c640f2764005ad7da9a823f494f10e639